src/Skoruba.IdentityServer4.Admin.BusinessLogic.Identity/Extensions/AdminServicesExtensions.cs

```diff
@@ -15,7 +15,7 @@
 using Skoruba.IdentityServer4.Admin.BusinessLogic.Identity.Services.Interfaces;
 using Skoruba.IdentityServer4.Admin.EntityFramework.Interfaces;
 
-namespace Skoruba.IdentityServer4.Admin.BusinessLogic.Identity.Extensions
+namespace Microsoft.Extensions.DependencyInjection
```

Comment:
I am not sure why you renamed this namespaces, but I think it should follow the project structure path like other namespaces do, otherwise it's going to lead to surprising behavior. (Now it looks like external library code.) Was it a requirement for registering something or similar?

Reply:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-2.2 - look at this please, this is a recommendation for extensions.